snowpack/src/commands/dev.ts

```diff
@@ -891,7 +891,7 @@ export async function startServer(
     // Defer "chokidar" loading to here, to reduce impact on overall startup time
     const chokidar = await import('chokidar');
     watcher = chokidar.watch(Object.keys(config.mount), {
-      ignored: config.exclude,
+      ignored: config.exclude.filter((k) => k !== '**/_*.{sass,scss}'), // Sass partials ignored for builds, but not for dev changes
```

Comment:
cc @FredKSchott this adds Sass partials back to the dev server watched files.

The alternative was a bit tricker: we are breaking on building Sass partials, not to mention wasting compute building files that are just supposed to be discarded. Though it seems a bit hacky, I think this is a more-correct approach here: ignore Sass partials by default, but watch for changes. IMO that‘s better than always build Sass partials, even though people don‘t want us to (also there‘s a bigger bug there).